components/organism/nestedCheckboxes/src/index.js

```diff
@@ -0,0 +1,117 @@
+import React, {useState} from 'react'
+import PropTypes from 'prop-types'
+
+import MoleculeCheckboxField from '@s-ui/react-molecule-checkbox-field'
+
+const baseClass = 'sui-OrganismNestedCheckboxes'
+
+const noop = () => {}
+
+const OrganismNestedCheckboxes = ({
+  id,
+  labelParent = '',
+  onChangeParent = noop,
```

Comment:
what means noop? 😅  IMO add here the default function improves the readability

Reply:
https://en.wikipedia.org/wiki/NOP_(code)

`noop` means something, so it's readable.

Reply:
Changed ;)